cmd/zdb/zdb.c

```diff
@@ -221,6 +221,7 @@ dump_debug_buffer(void)
 {
 	if (dump_opt['G']) {
 		(void) printf("\n");
+		(void) fflush(stdout);
 		zfs_dbgmsg_print("zdb");
```

Comment:
nit: we should add an empty line after dumping the debug buffer for `zdb` and `ztest` to make it easier to read.  Right now, it's easy for the ASSERT to blend in to the previous log.  For example:

```
...
spa_misc.c:409:spa_load_note(): spa_load(ztest_CHECKPOINTED_UNIVERSE, config trusted): LOADED
spa.c:7098:spa_async_request(): spa=ztest_CHECKPOINTED_UNIVERSE async request task=32
spa_open(ztest_opts.zo_pool, &spa, FTAG) == 0 (0x6 == 0)
ASSERT at ztest.c:6834:ztest_run()/home/behlendo/src/git/zfs/cmd/ztest/.libs/lt-ztest[0x409eac]
/lib64/libpthread.so.0(+0xf6d0)[0x7f50873426d0]
/lib64/libc.so.6(gsignal+0x37)[0x7f5086f9c277]
/lib64/libc.so.6(abort+0x148)[0x7f5086f9d968]
/home/behlendo/src/git/zfs/cmd/ztest/.libs/lt-ztest[0x40aff1]
/home/behlendo/src/git/zfs/cmd/ztest/.libs/lt-ztest[0x408de7]
/lib64/libc.so.6(__libc_start_main+0xf5)[0x7f5086f88445]
/home/behlendo/src/git/zfs/cmd/ztest/.libs/lt-ztest[0x409502]

ZFS_DBGMSG(ztest):
spa.c:4429:spa_open_common(): spa_open_common: opening ztest
spa_misc.c:409:spa_load_note(): spa_load(ztest, config trusted): LOADING
...
```